nx-X11/programs/Xserver/os/io.c

```diff
@@ -965,6 +965,10 @@ WriteToClient (ClientPtr who, int count, const void *__buf)
 	  CriticalOutputPending = FALSE;
 	  NewOutputPending = FALSE;
 	}
+
+	if (FlushCallback)
+	    CallCallbacks(&FlushCallback, NULL);
+
 	return FlushClient(who, oc, buf, count);
```

Comment:
This commit has been removed now and will appear as reference in  65b6a62bc726b5e4f4833ed1e27733e22fabcfa8.